nixos/modules/services/audio/upmpdcli.nix

```diff
@@ -0,0 +1,88 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+with pkgs;
+with lib;
+
+let
+  uid = config.ids.uids.upmpdcli;
+  gid = config.ids.gids.upmpdcli;
```

Comment:
I don't think these exists, see `nixos/modules/misc/ids.nix`. I see you are setting `DynamicUser = true` as well, could you try getting it to work with that instead?